appyters/ChIP_seq/appyter.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://raw.githubusercontent.com/MaayanLab/appyter-catalog/master/schema/appyter-validator.json",
+  "name": "ChIP_seq",
+  "title": "ChIP-seq analysis pipeline",
+  "version": "0.0.1",
+  "description": "ChIP-seq analysis pipeline enables you to analyze and visualize datasets with state-of-the-art algorithms and visualization methods.",
+  "image": "screenshot.png",
+  "authors": [
+    {
+      "name": "Minji Jeon",
+      "email": "minji.jeon1@gmail.com"
+    }
+  ],
+  "url": "https://github.com/MaayanLab/appyter-catalog",
+  "tags": [
+    "ChIP-seq", "RNAseq"
```

Comment:
In the future, please review the tags already on the catalog. I'll be changing this to `RNA-seq`

Reply:
Got it!

Reply:
Can I keep ChIP-seq?

Reply:
Yeah ChIP-seq is fine